www/js/banidb/realm-search.js

```diff
@@ -1,7 +1,29 @@
+/* eslint-disable import/no-dynamic-require, global-require */
+const electron = require('electron');
+const path = require('path');
 const Realm = require('realm');
-const realmDB = require('./realm-db');
+
 const CONSTS = require('./constants');
 
+const { remote } = electron;
+const userDataPath = remote.app.getPath('userData');
+const realmPath = path.resolve(userDataPath, 'sttmdesktop-evergreen.realm');
+const realmSchemaPath = path.resolve(userDataPath, 'realm-schema-evergreen.json');
+
+const realmSchemaDef = {
```

Comment:
nit picking- is this the appropriate name? this object is more than just the schemaDef, its all the config you can pass to the `open` function.
https://realm.io/docs/javascript/latest/api/Realm.html#.open

Reply:
We can make it `realmSchemaConfig`